### PR TITLE
Database

### LINKED ISSUE #33: Database
# User-Story

As a user

I want to be able to  upload projects

# Notes, Links, Material

https://www.npmjs.com/package/json-server

# ToDos

- [x] Install json server
- [x] Add db.json
- [ ] Configure routes (upload, search/filter, delete?)

### PR DESCRIPTION
**Pull request by kschmitz123**
Latest projects are displayed on the starting page. Projects can be browsed by categories.

Todo
Search function will be implemented later.

close #33

### CODE REVIEW

**Review by Sina-Sche — APPROVED** [review]
lgtm 💯  Very impressive!! 🤯

File: client/src/utils/api.js [hunk 1]
```diff
@@ -13,3 +13,19 @@ export async function getProjectById(id) {
...
+}
+export async function getLimitedProjects() {
+  const response = await fetch("/api/projects?_sort=id&_order=desc&_limit=20");
+  const projects = await response.json();
+  return projects;
+}
+export async function getCategory(category) {
+  const response = await fetch(`/api/projects?data.category=${category}`);
+  const projects = await response.json();
+  return projects;
```
Comment (AlexisRoe) [inline client/src/utils/api.js]: mabye making one function

query = "?_sort=id&_order=desc"
query = "?_sort=id&_order=desc&_limit=20"
query =  "?data.category=${category}"

```suggestion
export async function getData (query) {
  const response = await fetch(`/api/projects${query}`);
  const data = await response.json();
  return data;
}
```

**Review by AlexisRoe — COMMENTED** [review]
a simpler more felxibler version for fetching data

**Comment by SvLowman** [comment]
👍

**Pull request merged**